pages/api/cities.js

```diff
@@ -0,0 +1,49 @@
+const handler = (req, res) => {
+  const cityLocations = [
+    { city: 'Москва', branch: 92 },
+    { city: 'Санкт-Петербург', branch: 16 },
+    { city: 'Калининград', branch: 10 },
+    { city: 'Красногорск', branch: 5 },
+    { city: 'Воронеж', branch: 4 },
+    { city: 'Долгопрудный', branch: 3 },
+    { city: 'Красноярск', branch: 3 },
+    { city: 'Раменское', branch: 3 },
+    { city: 'Пушкино', branch: 2 },
+    { city: 'Гатчина', branch: 2 },
+    { city: 'Новороссийск', branch: 2 },
+    { city: 'Балашиха', branch: 2 },
+    { city: 'Воскресенск', branch: 2 },
+    { city: 'Рязань', branch: 2 },
+    { city: 'Котельники', branch: 2 },
+    { city: 'Кемерово', branch: 2 },
+    { city: 'Орехово-Зуево', branch: 2 },
+    { city: 'Симферополь', branch: 2 },
+    { city: 'Дербент', branch: 2 },
+    { city: 'Казань', branch: 2 },
+    { city: 'Тюмень', branch: 2 },
+    { city: 'Жуковский', branch: 2 },
+    { city: 'Псков', branch: 2 },
+    { city: 'Ростов-на-Дону', branch: 2 },
+    { city: 'Тамбов', branch: 2 },
+    { city: 'Хабаровск', branch: 2 },
+    { city: 'Норильск', branch: 2 },
+    { city: 'Белгород', branch: 2 },
+    { city: 'Щелково', branch: 2 },
+    { city: 'Краснодар', branch: 2 },
+    { city: 'Нижний Новогород', branch: 2 },
+    { city: 'Сочи', branch: 2 },
+    { city: 'Химки', branch: 2 },
+    { city: 'Североморск', branch: 1 },
+    { city: 'Рославль', branch: 1 },
+    { city: 'Старый Оскол', branch: 1 },
+    { city: 'Прохладный', branch: 1 },
+    { city: 'Чита', branch: 1 },
+    { city: 'Электрогорск', branch: 1 },
+  ];
+
+  res.status(200).json({
+    message: 'Cities List!',
```

Comment:
As API developer, I will tell you that one of the key principles of API is a consistency of interfaces, so in this case something like `success: true` will bring more help